src/main/java/net/dries007/holoInventory/compat/InventoryDecoderRegistry.java

```diff
@@ -80,6 +83,28 @@ public NBTTagList toNBT(IInventory inv) {
                 }
             });
         }
+        if (Loader.isModLoaded("gregtech")) {
+
+            REGISTERED_INVENTORY_DECODERS.add(new InventoryDecoder(BaseMetaTileEntity.class) {
+
+                @Override
+                public NBTTagList toNBT(IInventory inv) {
+                    IMetaTileEntity metaTileEntity = ((BaseMetaTileEntity) inv).getMetaTileEntity();
+                    if (!(metaTileEntity instanceof GT_MetaTileEntity_QuantumChest)) {
+                        return DEFAULT.toNBT(inv);
+                    }
+                    NBTTagList list = new NBTTagList();
+                    ItemStack stack = inv.getStackInSlot(1);
+                    if (stack != null) {
+                        NBTTagCompound tag = stack.writeToNBT(new NBTTagCompound());
+                        int item_amount = ((GT_MetaTileEntity_QuantumChest) metaTileEntity).mItemCount;
+                        tag.setInteger(NBT_KEY_COUNT, stack.stackSize + item_amount);
```

Comment:
This can cause integer overflow. Set separated tag for `inv.getStackInSlot(1)` and `mItemCount`, they will be merged later if necessary